registry/auth/token/token_test.go

```diff
@@ -2,115 +2,127 @@ package token
 
```

Comment:
I'd be tempted to make this two commits, the first one for the implementation, the second for the tests. That way we know we pass the current tests before changing them to use the new library.

Reply:
Funny you should mention that - that's literally how I went about it 😄 I started with the implementation first, got the tests passing with `libtrust` module, then modified the tests. Happy to do this in 2 steps if folks want me to.

Reply:
Might be a good proof point, but I'm happy if you say you did it :)